src/components/frame/LandingPage/LandingPage.tsx

```diff
@@ -51,19 +82,11 @@ const LandingPage = () => {
           />
         </div>
         <ul>
+          {wallet && user === null && (
```

Comment:
I have added a `null` option to the AppContext's user state, i.e. it can now either be `undefined`, `null`, or `User` (as opposed to `undefined` or `User`). This gives us a way of distinguishing between the case where there is no user registered, and the case where we are still in the process of determining if there is a user registered (previously both would've been `undefined`). 

I was using `!userAccountRegistered`, but the `useUserAccountRegistered` hook returns a bool, and thus isn't flexible enough to indicate the distinction outlined above. Without it, the Landing page will always render the `Create a username` item, before rendering the `Create a colony` item if appropriate (which leads to ui flicker). 

This way it only loads the correct item, not one then the other straight after.

Reply:
Could you not do this check using `userLoading` from `AppContext`?
```
(!userLoading && !user)
```
I don't think your approach is wrong (and it still might make sense to allow `User` to be null), just wondering if this would be a bit clearer about your intention, which I understand as "if the loading is finished and there's still no user"

Reply:
@jakubcolony You're right. Thanks! I will revert the introduction of a null state given it's no longer relevent to this pr. It may be reintroduced if it's material to Olga's user settings pr or similar.

Reply:
Cool, so this is actually implemented, so that @chinins can pull in your changes directly

Reply:
Ok, will leave the addition of null as is inside AppContext, but change check in `Landing` as Jakub suggests for readablility